### PR TITLE
fix NestStaking contract and DeployNestContracts script

### PR DESCRIPTION
**Pull request by ungaro**
## What's new in this PR?

fix NestStaking contract and DeployNestContracts to use ComponentToken instead of FakeComponentToken

### CODE REVIEW

File: nest/script/UpgradeNestContracts.s.sol
Comment (eyqs) [inline nest/script/UpgradeNestContracts.s.sol:1]: Added this upgrade script

File: nest/src/AggregateToken.sol [hunk 1]
```diff
@@ -221,7 +221,8 @@ contract AggregateToken is ComponentToken, IAggregateToken, ERC1155Holder {
 
     /**
      * @notice Request to sell ComponentToken.
-     * @dev Only the owner can call this function. This function requests the sale of ComponentToken, which will be processed later.
+     * @dev Only the owner can call this function. This function requests the sale of ComponentToken, which will be
```
Comment (eyqs) [inline nest/src/AggregateToken.sol:224]: i called `forge fmt`

**Review by ungaro — COMMENTED** [review]
lgtm!

**Review by eyqs — APPROVED** [review]
LGTM

**Pull request merged**